src/backend/distributed/sql/udfs/enable_citus_mx_for_pre_citus11/latest.sql

```diff
@@ -0,0 +1,35 @@
+CREATE OR REPLACE FUNCTION pg_catalog.enable_citus_mx_for_pre_citus11()
```

Comment:
We usually start operational functions with citus_

also wonder whether we should mirror citus_finish_pg_upgrade (e.g. citus_finish_upgrade_to_citus11)

Reply:
also given that you don't have call this right after upgrade to 11 (e.g., say you upgrade from 10 -> 12), we should probably avoid `11` in the name.